articles/azure-monitor/app/how-do-i.md

```diff
@@ -131,16 +131,25 @@ Learn more about [pricing and quotas](../../azure-monitor/app/pricing.md).
 ## Disable telemetry
 To **dynamically stop and start** the collection and transmission of telemetry from the server:
 
-```
+### ASP.NET Classic applications
 
+```csharp
     using  Microsoft.ApplicationInsights.Extensibility;
 
     TelemetryConfiguration.Active.DisableTelemetry = true;
 ```
 
+### Other applications
+It is not recommended to use `TelemetryConfiguration.Active` singleton on console or ASP.NET Core applications.
+if you created `TelemetryConfiguration` instance yourself - set `DisableTelemetry` to `true`.
+
+For ASP.NET Core applications you may access `TelemetryConfiguration` instance using [ASP.NET Core dependency injection](https://docs.microsoft.com/en-us/aspnet/fundamentals/dependency-injection)
```

Comment:
also link to asp.net core ai sdk page : https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core